packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,29 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    instance = this
+    subscription = instance.subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName.value
+        jobTitle: form.jobTitle.value
+        bio: form.bio.value
+        emailHidden: form.emailHidden.value
+      }
+      Meteor.call('updateProfile', form.profileId.value, fields, -> {})
+
+if Meteor.isServer
+  Meteor.methods
+    updateProfile: (profileId, fields, callback) ->
+      userProfile = UserProfiles.findOne({_id: profileId})
+      userProfile.update fields, callback
```

Comment:
What are the circumstances where you might be able to edit someone else's profile?  I only split out the UserProfile model from the Meteor Users collection so that we could get more custom behavior.  I was under the impression we wanted to think of Users and Profiles as a 1-to-1 relationship.